llvm/test/CodeGen/RISCV/GlobalISel/irtranslator/vec-ld.ll

```diff
@@ -0,0 +1,7 @@
+; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+; RUN: llc -mtriple=riscv32 -mattr=+v -global-isel -stop-after=irtranslator -verify-machineinstrs < %s | FileCheck  -check-prefixes=RV32I %s
+
+define void @vload_vint8m1(ptr %pa) {
```

Comment:
I think this is missing test checks. Also, I think we need to test a more complete set of scalable vector types. Lastly, I think we need test checks for rv64 as well.

Reply:
Sure, I'll add more tests with different types and to include rv64 if this simplest test case makes sense.
Right now, I'm wondering, when I use `llvm/utils/update_mir_test_checks.py `, the `vec-ll.s` file auto-generate `align 8` at the end of `%va = load <vscale x 8 x i8>, ptr %pa`, so I appended it. Does it make sense to add `align 8`?

Reply:
The alignment is relative to EEW not XLen.

> If an element accessed by a vector memory instruction is not naturally aligned to the size of the element, either the element is transferred successfully or an address misaligned exception is raised on that element.